src/components/FreeTrialEulas.js

```diff
@@ -0,0 +1,1338 @@
+import { Component } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { Markdown } from 'src/components/common'
+
+const broadEula = `
+1. The Terra Free Credits Program (“credits”), sponsored by Google, is administered by [Onix Networking](https://www.onixnet.com/products/google-cloud/google-cloud-platform/google-app-engine) (“Onix”), a Google Cloud Premier Partner. 
+
+2. By opting into this program, you are authorizing Terra to give Onix and Google access to your Terra user profile information. This is necessary for Onix and Google to give you the free credits. 
+
+3. Your credits of $300 will expire December 30, 2018 or 60 days after they were issued, whichever comes first.
```

Comment:
Note that Dec 30, 2018 has already passed. I would confirm with @tmm211 or Ilyana about whether these are the actual terms of the agreement with Onix.

Reply:
Thanks for noticing this! I checked with Tiffany and the new date is just Dec 30, 2019